config/lnl/10-platform.toml

```diff
@@ -0,0 +1,57 @@
+version = [3, 0]
```

Comment:
Ehm, not following why we have a 10 prefix in our filenames. I want to #include these via the pre-processor.